proto/agent.proto

```diff
@@ -168,6 +168,13 @@ enum ContainerHealth {
 	unhealthy = 3;
 }
 
+// ContainerNetworkAddress records the IPs, Ports and Protocols for each container
+message ContainerNetworkAddress {
+	string ip = 1;
```

Comment:
Since this is repeated inside container, when will a container have multiple IPs ?

Reply:
We spoke offline, a container will have a different ip if it is assigned a different network namespace